trimmomatic/0.39/Dockerfile

```diff
@@ -1,11 +1,14 @@
+ARG TRIMMOMATIC_VER="0.39"
+
 #use ubuntu as base image
 FROM ubuntu:xenial as app
+ARG TRIMMOMATIC_VER
 
 # metadata
 LABEL base.image="ubuntu:xenial"
 LABEL version="1"
```

Comment:
trimmomatic is an "older" container, so it's missing some of the labels that we now "recommend"

Could you add 
```
LABEL dockerfile.version="2"
LABEL license.url="https://github.com/cov-lineages/pangolin/blob/master/LICENSE.txt"
```
And add yourself as a maintainer?